static/app/views/dashboardsV2/widgetBuilder/buildSteps/visualizationStep.tsx

```diff
@@ -20,6 +21,25 @@ import {displayTypes} from '../utils';
 
 import {BuildStep} from './buildStep';
 
+const DisplayOptionLabel = styled('span')`
+  display: flex;
+  justify-content: space-between;
+  width: calc(100% - ${space(1)});
+`;
+
+const DISPLAY_OPTIONS = Object.keys(displayTypes).map(value => ({
+  label:
+    value === DisplayType.TOP_N ? (
+      <DisplayOptionLabel>
+        {displayTypes[value]}
+        <Tag type="info">{t('deprecated')}</Tag>
```

Comment:
I just realized that we should feature flag this against `new-widget-builder-experience-design` because we only want it available with the new sort by field

Reply:
👍  ah good call, I've added the `new-widget-builder-experience-design` feature flag to the conditions!